app/theme.ts

```diff
@@ -46,6 +58,11 @@ const main: MainTheme = {
     body: raleway,
     title: oswald,
   },
+
+  device: {
```

Comment:
Good point, I think I named it devices because there's a `sizes` property which contains `maxWidth`. Do you think it should still be renamed to `sizes` or could something like `screenSizes` work?